src/index.ts

```diff
@@ -78,9 +81,45 @@ async function run() {
     process.exit(1);
   }
 
+  if (!program.framework) {
+    const res: prompts.Answers<string> = await prompts({
+      choices: [
+        { title: 'React', value: 'react' },
+        { title: 'Vue', value: 'vue' }
+      ],
+      message: "Choose a front end framework:",
+      name: "framework",
+      type: "select",
+    });
+
+    if (typeof res.framework === "string") {
+      framework = res.framework.trim();
+    }
```

Comment:
Move this to the outside of this control structure since we only need to `trim` custom user input